storage/ndb/rest-server/rest-api-server/internal/testutils/schemata.go

```diff
@@ -42,7 +42,7 @@ func CreateDatabases(
 		err = runQueries(createSchema, dbConn)
 		if err != nil {
 			cleanupDbsWrapper(dropDatabases)()
-			err = fmt.Errorf("failed running createSchema for db '%s'; error: %w", db, err)
+			err = fmt.Errorf("failed running createSchema for db '%s'; error: %v", db, err)
```

Comment:
Go has a special directive for errors when formatting strings `%w`. You can basically unwrap the errors later layer by layer if you use it.

Reply:
reverted